test/WebSites/ModelBindingWebSite/Controllers/VehicleController.cs

```diff
@@ -59,7 +59,6 @@ public IActionResult UpdateDealerVehicle(VehicleWithDealerViewModel model)
                 return PartialView("UpdateVehicle", model);
             }
 
-            model.Update();
```

Comment:
The code basically did nothing and sort of tested that services could be property injected.